gen/flights/flightplan.py

```diff
@@ -114,6 +116,65 @@ def tot_waypoint(self) -> Optional[FlightWaypoint]:
         failed to generate. Nevertheless, we have to defend against it.
         """
         raise NotImplementedError
+    
+    @cached_property
+    def bingo_fuel(self) -> int:
+        """Bingo fuel value for the FlightPlan
+        """
+        if self.waypoints is None or self.farthest_wpt_from_arrival is None:
+            return 0
+
+        distanceToArrival = self.farthest_wpt_from_arrival.position.distance_to_point(self.flight.arrival.position)
+        distanceToArrival = meter_to_nm(distanceToArrival)
+
+        bingo = 1000 # Minimum Emergency Fuel
+        bingo += 500 # Visual Traffic
+        bingo += 15 * distanceToArrival
+
+        if self.flight.divert is not None and self.farthest_wpt_from_divert is not None:
+            assert self.farthest_wpt_from_divert is not None
+            assert self.flight.divert is not None
+            distanceToDivert = self.farthest_wpt_from_divert.position.distance_to_point(self.flight.divert.position)
+            distanceToDivert = meter_to_nm(distanceToDivert)
+            bingo += 10 * distanceToDivert
```

Comment:
Am I reading this correctly? Having a divert option increases your bingo?

Reply:
Yes.

Reply:
That doesn't make sense to me. Why does having more options mean you need to RTB sooner?